src/NopeString.ts

```diff
@@ -114,6 +114,25 @@ export class NopeString extends NopePrimitive<string> {
     return this.test(rule);
   }
 
+  public between(
+    startLength: number,
+    endLength: number,
+    atLeastMessage = 'Input is too short',
+    atMostMessage = 'Input is too long',
+  ) {
+    if (startLength && endLength && startLength > endLength) {
+      const rule: Rule<unknown> = () => {
+        throw Error(
+          'between must receive an initial length (startLength) smaller than the final length (endLength) parameter',
+        );
+      };
+
+      return this.test(rule);
+    }
+
+    return this.atLeast(startLength, atLeastMessage) && this.atMost(endLength, atMostMessage);
```

Comment:
I know it isn't documented (should be done, I've put it on the todo list), but the correct way to reuse the existing validation methods in the validator should be to call them separately and return the instance back

This will essentially do the same, but IMO will be much more understandable:

Instead of 
```js
return this.atLeast(startLength, atLeastMessage) && this.atMost(endLength, atMostMessage);
```

Simply do
```js
this.atLeast(startLength, atLeastMessage);
this.atMost(endLength, atMostMessage);

return this;
```

This is more in-line with how the idea of the project currently works (calling validation methods to push rules on to the validation queue)